packages/smart-file-system/README.md

```diff
@@ -158,3 +158,6 @@ $smartFileInfos = $finderSanitizer->sanitize($files);
 // always array of Symplify\SmartFileSystem\SmartFileInfo
 var_dump($smartFileInfos);
 ```
+## Contribute
+
+The sources of this package are contained in the symplify monorepo. We welcome contributions for this package on [symplify/packages/smart-file-system](https://github.com/symplify/symplify/tree/master/packages/smart-file-system).
```

Comment:
Just curious... is this in each package's README.md? If not, it should be to be consistent

Reply:
I copied it over from https://github.com/migrify/config-transformer/blob/master/README.md

It's certainly a good idea if you want to create some sort of template for this; but the link `https://github.com/symplify/symplify/tree/master/packages/smart-file-system` is different in every package.

Reply:
There should be just one link in `symplify/symplify`, no need to handle it per package

Reply:
Well, better than nothing ;-) But I don't know how to setup such a template - so it's up to you...

Reply:
No need to setup template :), it's 2 mins work. 

Just add the same text with link to 1 repository - https://github.com/symplify/symplify, to all packages' `README.md`.